.github/workflows/reusable_package_mage.yaml

```diff
@@ -0,0 +1,180 @@
+name: "Reusable package mage and upload"
+on:
+  workflow_call:
+    inputs:
+      arch:
+        type: string
+        description: "Architecture to build the image for"
+        required: true
+      mage_version:
+        type: string
+        description: "Mage version built into this image (format: X.Y.Z)"
+        required: true
+      mage_build_scope:
+        type: string
+        description: "Mage build scope"
+        default: 'all'
+      mage_build_target:
+        type: string
+        description: "Mage build target"
+        default: 'prod'
+      memgraph_version:
+        type: string
+        description: "Memgraph version built into this image (format: X.Y.Z)"
+        required: true
+      memgraph_download_link:
+        type: string
+        description: "Memgraph package download link. Leave empty to use the official download link."
+        default: ""
+      shorten_tag:
+        type: string
+        description: "Make the final tag as short as possible, shortest format is X.Y-memgraph-X.Y"
+        default: true
+      force_release:
+        type: string
+        description: "Overwrite existing image on dockerhub"
+        default: true
+      push_to_dockerhub:
+        type: string
+        description: "Push the image to DockerHub?"
+        default: false
+      push_to_s3:
+        type: string
+        description: "Push the image to S3?"
+        default: false
+
+env:
+  OS: ${{ inputs.arch == 'arm' && 'debian-11-aarch64' || 'debian-11' }}
```

Comment:
How does this check work? inputs.arch must be 'arm'  we 'debian-11-aarch64', but with what is 'debian-11-aarch64' compared

Reply:
This is a very specific way of writing a ternary expression for github actions. Let me explain what it does using a much better syntax, python code :joy: 
if inputs.arch == 'arm':
  OS = 'debian-11-aarch64'
else:
  OS = 'debian-11'